audio.py

```diff
@@ -88,3 +135,35 @@ def compute_duration(audio_path, backend = 'ffmpeg'):
 	cmd = ['ffprobe', '-v', 'error', '-show_entries', 'format=duration', '-of', 'default=noprint_wrappers=1:nokey=1'
 			] if backend == 'ffmpeg' else ['soxi', '-D'] if backend == 'sox' else None
 	return float(subprocess.check_output(cmd + [audio_path]))
+
+
+def timeit(audio_path, numbers, sample_rate, mono, audio_backend, delimiter):
```

Comment:
delimiter isn't a good name here. We use `scale` for this: https://github.com/vadimkantorov/convasr/blob/master/metrics.py#L59 and https://github.com/vadimkantorov/convasr/blob/master/models.py#L615

Reply:
done